### PR TITLE
Apply updated PPT

### PR DESCRIPTION
**Pull request by jeremykubica**
Mechanical application of PPT

### CODE REVIEW

File: src/dask_nested/example_benchmarks.py
Comment (dougbrn) [inline src/dask_nested/example_benchmarks.py:1]: As in the nested-pandas PR, this should be removed

File: docs/pre_executed/README.md
Comment (dougbrn) [inline docs/pre_executed/README.md:1]: this should be removed

File: docs/notebooks/README.md
Comment (dougbrn) [inline docs/notebooks/README.md:1]: As in the nested-pandas PR, this should be removed

**Review by dougbrn — CHANGES_REQUESTED** [review]
thank you! This looks good, just requesting a few things not be added as was also the case with the nested-pandas PR

File: src/dask_nested/example_benchmarks.py
Comment (jeremykubica) [reply src/dask_nested/example_benchmarks.py:1]: Done

File: docs/notebooks/README.md
Comment (jeremykubica) [reply docs/notebooks/README.md:1]: Done

File: docs/pre_executed/README.md
Comment (jeremykubica) [reply docs/pre_executed/README.md:1]: Done

**Review by dougbrn — APPROVED** [review]
Thank you!

**Pull request merged**